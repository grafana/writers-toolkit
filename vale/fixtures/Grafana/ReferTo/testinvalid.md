See [here](https://example.com).
Check out [the link](https://example.com).

For more information, see [link text](https://example.com).
For more information, check out [link text](https://example.com).
