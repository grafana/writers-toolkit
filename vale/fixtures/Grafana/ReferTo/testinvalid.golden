testinvalid.md:1:1:Grafana.ReferTo:When linking in Markdown, use 'Refer to [' instead of 'See ['.
testinvalid.md:2:1:Grafana.ReferTo:When linking in Markdown, use 'Refer to [' instead of 'Check out ['.
testinvalid.md:4:23:Grafana.ReferTo:When linking in Markdown, use 'refer to [' instead of 'see ['.
testinvalid.md:5:23:Grafana.ReferTo:When linking in Markdown, use 'refer to [' instead of 'check out ['.
